reoptjl/test/posts/all_inputs_test.json

```diff
@@ -145,7 +145,8 @@
         "emissions_factor_CO2_decrease_fraction": 0.01174,
         "emissions_factor_NOx_decrease_fraction": 0.01174,
         "emissions_factor_SO2_decrease_fraction": 0.01174,
-        "emissions_factor_PM25_decrease_fraction": 0.01174
+        "emissions_factor_PM25_decrease_fraction": 0.01174,
```

Comment:
add other new inputs (renewable_energy_fraction_series and include_grid_renewable_fraction_in_RE_constraints) to this post too?

Reply:
added in [fefcfd4](https://github.com/NREL/REopt_API/pull/628/commits/fefcfd46935d02a16949db5cce286209604cc068)